Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 # Note: some of these values are also used when building Debian packages below.
 name = "krill"
-version = "0.15.0-dev"
+version = "0.14.3"
```

Comment:
Shouldn't this be left as 0.15.0-dev?

Reply:
Perhaps it would be more clean to make this change in a separate PR. If you prefer I can do that. But, this is going to be in a bug fix release, so 0.14.3 is more appropriate than 0.15.0.